src/ToDo.UI/Views/HomePage.xaml

```diff
@@ -68,67 +69,68 @@
 			<muxc:NavigationView.PaneHeader>
 				<Grid Grid.Row="0"
 					  Padding="16,8"
-					  ColumnSpacing="8">
-					<Grid.ColumnDefinitions>
-						<ColumnDefinition Width="Auto" />
-						<ColumnDefinition Width="*" />
-						<ColumnDefinition Width="Auto" />
-					</Grid.ColumnDefinitions>
-
-					<PersonPicture Grid.Column="0"
-								   DisplayName="{Binding CurrentUser.Name}"
-								   d:ProfilePicture="{Binding CurrentUser.AvatarUrl}"
-								   Width="40"
-								   Height="40"
-								   VerticalAlignment="Center" />
-
-					<TextBlock Grid.Column="1"
-							   Text="{Binding CurrentUser.Name}"
-							   VerticalAlignment="Center" />
-
-					<!-- This is a hack to allow both picture and textblock to respond to tap event -->
-					<Border Background="Transparent"
+					  ColumnSpacing="8"
+					  ColumnDefinitions="*,AUto">
+					
+					<Button Grid.Column="0"
```

Comment:
@carldebilly These are no longer figma generated code, but is necessary to get the desired result.
Do we accept this, and make change to the figma design + plugin to codegen this?

Reply:
The previous code wasn't generated anyway.